packages/babel-traverse/test/removal.js

```diff
@@ -33,4 +33,15 @@ describe("removal", function() {
       expect(generateCode(rootPath)).toBe("x = () => {};");
     });
   });
+
+  it("remove with noScope", function() {
+    const ast = parse("a=1");
+    traverse(ast, {
+      AssignmentExpression: function(path) {
+        path.remove();
+      },
```

Comment:
I guess we should add `noScope: true` here?

Reply:
Yes, I'm a fool...